eth/config.go

```diff
@@ -71,8 +71,7 @@ func init() {
 	}
 }
 
-//go:generate gencodec -type Config -field-override configMarshaling -formats toml -out gen_config.go
-
+//Config is go:generate gencodec -type Config -field-override configMarshaling -formats toml -out gen_config.go
```

Comment:
Hah no, this is a special statement that is interpreted by `go generate` to generate source files. Please don't modify this.

Reply:
thanks ill change it back